pkg/agent/controller/networkpolicy/audit_logging.go

```diff
@@ -143,10 +145,11 @@ func newAntreaPolicyLogger() (*AntreaPolicyLogger, error) {
 		Compress:   true, // compress the old log files for backup
 	}
 
-	cAntreaPolicyLogger := &AntreaPolicyLogger{
+	antreaPolicyLogger := &AntreaPolicyLogger{
+		bufferLength:     time.Second,
 		anpLogger:        log.New(logOutput, "", log.Ldate|log.Lmicroseconds),
 		logDeduplication: logRecordDedupMap{logMap: make(map[string]*logDedupRecord)},
 	}
-	klog.V(2).Infof("Initialized Antrea-native Policy Logger for audit logging with log file '%s'", logFile)
-	return cAntreaPolicyLogger, nil
+	klog.InfoS("Initialized Antrea-native Policy Logger for audit logging with log file '%s'", logFile)
```

Comment:
```suggestion
	klog.InfoS("Initialized Antrea-native Policy Logger for audit logging", "logFile", logFile)
```

structured logging means that some of the information is organized as Key-Value data, instead of having a flat message string